hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -243,6 +243,10 @@ public static boolean isReadOnly(
     case ListKeys:
     case ListKeysLight:
     case ListTrash:
+      // This is deprecated by HDDS-11251. Keeping this in here
```

Comment:
For me it is not clear if this comment is talking about ListTrash, or ServiceList. Can we write ListTrash is deprecated... instead? I know if one gets the git annotate output, or if one checks the JIRA then it is clear what the comment is talking about, but that requires an extra hop, so I would like to make our life easier later with this change. Same goes to the other comment for RecoverTrash.

Reply:
Addressed in the latest commit [3172f3c](https://github.com/apache/ozone/pull/7060/commits/3172f3c9f13cfc3d4779871bb8e484f0aca889c5)